src/MarginalSamplingInterpreter.cc

```diff
@@ -21,14 +21,15 @@
  * see <http://www.lsstcorp.org/LegalNotices/>.
  */
 
-#include "boost/math/special_functions/round.hpp"
-
 #include "ndarray/eigen.h"
 
-#include "lsst/meas/modelfit/MarginalSamplingInterpreter.h"
+#include <cmath>
+
```

Comment:
Shouldn't go below the ndarray include.

Overall, it seems like the order of headers here was fine before, and now it's generally messed up.  Maybe it'd be better to just rebase away the last commit?